### DISCUSSION TITLE
Voting

### DISCUSSION THREAD

**Discussion by vsstudios** [sandarutharuneth/ivongiveaways#1] [Q&A] [answered]
What happen when we vote for the bot? Is it gonna add extra points for running giveaway?

**Comment by sandarutharuneth** [comment, marked as answer]
> What happen when we vote for the bot? Is it gonna add extra points for running giveaway?

Yes but you have to be on a valid giveaway. Bot will automatically add some extra points for voting.

**Reply by vsstudios** [reply]
Is there an amount of points?

**Reply by sandarutharuneth** [reply]
Nope. It's random
